docs/windows.md

```diff
@@ -171,18 +171,52 @@ curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/lates
 .\PrepareNode.ps1 -KubernetesVersion v1.18.0
 ```
 
-4. Prepare network adapter for kube-proxy
+4. Prepare node environment needed by antrea-agent
 
-kube-proxy needs a network adapter to configure Kubernetes Service IPs and uses
-the adapter for proxying connections to Service. Use following script to create the network
-adapter.
+Run following commands to prepare node environment needed by antrea-agent:
 ```
+mkdir c:\k\antrea
+cd c:\k\antrea
+curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Clean-AntreaNetwork.ps1
 curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Prepare-ServiceInterface.ps1
```

Comment:
should we merge `Prepare-ServiceInterface.ps1` into `Prepare-AntreaAgent.ps1`?

Reply:
Good question. It's better to move the content of `Prepare-ServiceInterface.ps1` in `Prepare-AntreaAgent.ps1` and remove previous script.

But `Prepare-ServiceInterface.ps1` is ready consumed by users. So what about keep the it for compatibility?

Reply:
I think it's ok to "break" things a bit here (the script would still be available for the v0.10.0 release tag after all), but I'll leave it up to you. I just think that reducing the number of overall steps is better.

Reply:
Thanks Antonin, let's keep it for now.